backup-loop.sh

```diff
@@ -244,6 +246,41 @@ restic() {
   call_if_function_exists "${@}"
 }
 
+rclone() {
+  _find_old_backups() {
+    command rclone lsf --format "tp" "${DESTINATION}/${BACKUP_NAME}*" | awk \
+            -v DESTINATION="${DESTINATION}" \
+            -v PRUNE_DATE="$(date '+%Y-%m-%d %H:%M:%S' --date="${PRUNE_BACKUPS_DAYS} days ago")" \
+            'BEGIN { FS=";" } $1 > PRUNE_DATE { printf "%s/%s\n", DESTINATION, $2 }'
+  }
+  init() {
+    if [ "${RCLONE_MULTIPATH,,}" != "true" ] && (( PRUNE_BACKUPS_DAYS > 0 )); then
+      log ERROR "Non-multipath RCLONE prunning is not supported"
+      exit 1
+    fi
+  }
+  backup() {
+    if [ "${RCLONE_MULTIPATH,,}" == "true" ]; then # TODO: Find a better name
```

Comment:
Yeah, I also feel like there's a better word than "mulitpath" but can only think of "additive" and "incremental" so far. Both of those are not quite right either since they imply different things in the context of backups.